articles/databox-online/azure-stack-edge-gpu-activation-key-vault.md

```diff
@@ -19,11 +19,12 @@ Azure Key Vault is integrated with Azure Stack Edge resource for secret manageme
 ## About Key vault and Azure Stack Edge
 
 Azure Key Vault cloud service is used to securely store and control access to tokens, passwords, certificates, API keys, and other secrets. Key Vault also makes it easy to create and control the encryption keys used to encrypt your data. 
+The pricing details of Azure Key Vault can be viewed [here](https://azure.microsoft.com/pricing/details/key-vault/).
```

Comment:
@shijojoy Could we change this to standard format: For more information, see the [Pricing details for Azure Key Vault](https://azure.microsoft.com/pricing/details/key-vault/).

Reply:
Made the change